unzipper.php

```diff
@@ -138,100 +145,111 @@ public static function extractGzipFile($archive, $destination) {
 }
 
 ?>
-
 <!DOCTYPE html>
-<head>
```

Comment:
Hmm, yoou seem to have some weird code format settings, or I had weird editor back in the days, but nevermind, will fix it later.